helm-charts/app-to-monitor/templates/deployment.yaml

```diff
@@ -68,6 +68,14 @@ spec:
             limits:
               cpu: "1"
               memory: "512Mi"
+          livenessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - "[ -f /tmp/integrity-monitor ] && pidof integritySum"
```

Comment:
Maybe we change the name of our app (integritySum)?

Reply:
This is possible, but, ok I'll do it configurable.